### DISCUSSION TITLE
Is there a configurable cache directory for pnpm?

### DISCUSSION THREAD

**Discussion by lancejpollard** [pnpm/pnpm#7618] [Q&A]
I am learning about Docker + AWS Lambda + EFS through https://docs.aws.amazon.com/lambda/latest/dg/nodejs-image.html#nodejs-image-clients and it says (about the Dockerfile contents):

```Dockerfile
# Required for Node runtimes which use npm@8.6.0+ because
# by default npm writes logs under /home/.npm and Lambda fs is read-only
ENV NPM_CONFIG_CACHE=/tmp/.npm
```

Is there any such thing to do for pnpm? I have been using pnpm but not sure if I need to switch back to npm for use in this Docker + AWS Lambda + EFS system.

Thanks for the help.

**Comment by woss** [comment]
referencing this, might be useful https://github.com/orgs/pnpm/discussions/4413
